versions/draft-3/SPEC.md

```diff
@@ -1018,12 +1018,31 @@ task memory_test {
 
 ```
 $parameter_meta = 'parameter_meta' $ws* '{' ($ws* $parameter_meta_kv $ws*)* '}'
-$parameter_meta_kv = $identifier $ws* '=' $ws* $string
+$parameter_meta_kv = $identifier $ws* ':' $ws* $expression
```

Comment:
This should probably be in the hermes grammar - IMO we're better off removing these opaque grammar blocks from the spec